pkg/ebpf/c/tracee.bpf.c

```diff
@@ -993,15 +1030,62 @@ BPF_HASH(module_init_map, u32, kmod_data_t, 256);                  // holds modu
 BPF_LRU_HASH(fd_arg_path_map, fd_arg_task_t, fd_arg_path_t, 1024); // store fds paths by task
 BPF_LRU_HASH(bpf_attach_map, u32, bpf_attach_t, 1024);             // holds bpf prog info
 BPF_LRU_HASH(bpf_attach_tmp_map, u32, bpf_attach_t, 1024);         // temporarily hold bpf_attach_t
-
+BPF_HASH(errors_count, bpf_error_t, bpf_error_count_t, 4096);      // errors count
```

Comment:
shouldn't this be a percpu map? or otherwise there can be race conditions, no?

Reply:
I think that my previous design was using percpu, but take a look in the errors_count entry:

```shell
❯ sudo bpftool map dump name errors_count | head -n13
[{
        "key": {
            "ret": 0,
            "id": "BPF_ERR_MAP_UPDATE_ELEM",
            "cpu": 2,
            "line": 3503,
            "file": "./pkg/ebpf/c/tracee.bpf.c"
        },
        "value": {
            "count": 3,
            "ts": 10421895290386
        }
    },{

```

Its key contains cpu, file and line, which I think make the entry to be accessed uniquely percpu; considering also that bpf doesn't preempt and that this map is not accessed from userspace. WDYT?

Reply:
I left some comments about atomicity in this map operations at https://github.com/aquasecurity/tracee/pull/2352#discussion_r1049162722. Yaniv is right, its either a PERCPU HASH or you have to use atomic operations for incrementing and zeroing.